### DISCUSSION TITLE
[IT] Add 'it/conforme/lineeGuidaAccessibilita'

### DISCUSSION THREAD

**Discussion by libremente** [publiccodeyml/publiccode.yml#105] [General]
# Context
Italian Public Administrations only 

# Details
Since Jan '20 it is necessary to attach a mandatory certification stating that the application/website is accessible (see https://docs.italia.it/AgID/documenti-in-consultazione/lg-accessibilita-docs/it/stabile/index.html in Italian language).
As such, I believe the `publiccode.yml` specification should provide the possibility to add this conformity. 

# Proposal
I suggest adding the following
```
key:  `it/conforme/lineeGuidaAccessibilita`
type: `boolean`
presence: `optional`

**Comment by bvhme** [comment]
Would it be possible to more generally list it as compliant with WCAG 2.1 (what I can understand from these guidelines) since this requirement is quite common under public administrations internationally.

**Comment by bfabio** [comment]
~~Posted to the forum as https://publiccode.discourse.group/t/add-it-conforme-lineeguidaaccessibilita-issue-83/64~~

Moved to GitHub Discussions

<sub>[Why?](https://github.com/publiccodeyml/publiccode.yml/discussions/104)</sub>
